src/content/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -312,6 +312,68 @@ newrelic-infrastructure:
   Setting `interval` to values larger than `40s` is not allowed.
 </Callout>
 
+### Filtering Namespaces [#filter-namespace]
+
+The Kubernetes Integration v3 and above allows filtering which namespaces will be scraped by allowing you to include or exclude namespaces based on labels. By default all namespaces will be scraped.
+
+In order to include only namespaces matching a certain label, change the `namespaceSelector` by adding the following to your `values-newrelic.yaml`, under the `newrelic-infratructure` section:
+
+```yaml
+common:
+  config:
+    namespaceSelector:
+      matchLabels:
+        newrelic.com/scrape: true
```

Comment:
This `true` value needs to be in string `"true"`. All the values both in `matchLabels` and `matchExpressions` need to be string or the integration will fail otherwise. Maybe I'd add a generic note as well explaining this. It can be confusing because of the usage of `"true"` but in the end could be anything as it's just a label. You can see this implemented in https://github.com/newrelic/nri-kubernetes/pull/495.